### ISSUE TITLE
Add react-addons-pure-render-mixin

### ISSUE THREAD

**Issue by Aweary** [developit/preact-compat#254]
We've got some older code that's still using `react-addons-pure-render-mixin`. It's a [really small module](https://github.com/facebook/react/blob/master/src/addons/ReactComponentWithPureRenderMixin.js), should be easy to just publish a standalone package and export it from `preact-compat/lib/ReactComponentWithPureRenderMixin`

> *Edit*: I published [standalone-react-addons-pure-render-mixin](https://www.npmjs.com/package/standalone-react-addons-pure-render-mixin)

**Comment by developit** [comment]
Open to a PR to re-export, or I can!

**Comment by mauron85** [comment]
There is another npm package https://www.npmjs.com/package/react-addons-pure-render-mixin published by gaeron.

Only exports are needed.

vi node_modules/react/lib/ReactComponentWithPureRenderMixin.js
```
module.exports = require('react-addons-pure-render-mixin');
```

**Comment by mauron85** [comment]
I was bit mistaken. This was already merged.

Not sure why, but same problem is with [preact-compat/react](https://github.com/preact-compat/react).

When you install it with:
```
npm i -S preact preact-compat
npm i -S preact-compat/react preact-compat/react-dom
```

I get webpack build error (probably same issue):

```
ERROR in ./~/react-addons-pure-render-mixin/index.js
Module not found: Error: Cannot resolve module 'react/lib/ReactComponentWithPureRenderMixin' in /home/finch/dev/mapilary/mapilary-tracker-ng/node_modules/react-addons-pure-render-mixin
 @ ./~/react-addons-pure-render-mixin/index.js 1:17-71
webpack built d3fa4c83b55a2b87fd82 in 16337ms
```

Fix is same as @aweary commit #255, but instead of preact-compat/lib you need to create react/lib/ReactComponentWithPureRenderMixin.js

One more remark. react-addons-pure-render-mixin seems not be compatible, but standalone-react-addons-pure-render-mixin definitely is.
